config/locales/cfp.fr.yml

```diff
@@ -0,0 +1,108 @@
+fr:
+  cfp:
+    abstract_hint: "Un petit paragraphe qui vous décrit succintement. (HTML)"
+    avatar_hint: "Uploader une nouvelle photo."
+    back: "Retours"
```

Comment:
Suspect plural.  Can you double-check please?

Reply:
You're right, I always write "Retour" "Retours", and that's a mistake.

Reply:
Done

Reply:
haha... missed this one :/